### PR TITLE
New Instrument Cal CSVs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
New instrument cal csvs created after vendor refurb.

### CODE REVIEW

File: calibration/SPKIRB/CGINS-SPKIRB-00275__20201124.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+275,CC_immersion_factor,"[1.368, 1.41, 1.365, 1.354, 1.372, 1.322, 1.347]", source_file SPKIR-B_OCR-507_SN_275_Calibration_Files_2020-11-24.zip > DI7275E.cal
+275,CC_offset,"[2147731779.6, 2147411938.1, 2147647189, 2147703545.6, 2147777807.6, 2147296455.8, 2147495144.2]",
+275,CC_scale,"[1.97843314677e-007, 2.01466552983e-007, 2.04358190893e-007, 2.05633621581e-007, 2.09476910922e-007, 2.08108508601e-007, 2.05353370429e-007]",
```
Comment (batryn) [inline calibration/SPKIRB/CGINS-SPKIRB-00275__20201124.csv:4]: Looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00271__20201124.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+271,CC_immersion_factor,"[1.368, 1.41, 1.365, 1.354, 1.372, 1.322, 1.347]", source_file SPKIR-B_OCR-507_SN_271_Calibration_Files_2020-11-24.zip > DI7271F.cal
+271,CC_offset,"[2147717714.9, 2147680233.8, 2147599778.3, 2148085174, 2147735838, 2147629051.7, 2147580058.8]",
+271,CC_scale,"[1.94060062952e-007, 2.21446039413e-007, 2.04285172528e-007, 2.14962122834e-007, 2.13312650986e-007, 2.13047409576e-007, 2.17956724947e-007]",
```
Comment (batryn) [inline calibration/SPKIRB/CGINS-SPKIRB-00271__20201124.csv:4]: Looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00270__20201124.csv [hunk 3]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+270,CC_immersion_factor,"[1.368, 1.41, 1.365, 1.354, 1.372, 1.322, 1.347]", source_file SPKIR-B_OCR-507_SN_270_Calibration_Files_2020-11-24.zip > DI7270F.cal
+270,CC_offset,"[2147466830.7, 2147460390.6, 2147271473, 2147812236.5, 2147674988.7, 2148419800.7, 2147541978.1]",
+270,CC_scale,"[2.2081728803e-007, 1.99458185736e-007, 2.03329575842e-007, 2.10906457451e-007, 2.21000324976e-007, 2.1479433564e-007, 2.22088253894e-007]",
```
Comment (batryn) [inline calibration/SPKIRB/CGINS-SPKIRB-00270__20201124.csv:4]: Looks good

File: calibration/PHSEND/CGINS-PHSEND-P0069__20210124.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0069,CC_ea434,17372.0,source_file PHSEN-D_SAMI-pH_SN_P0069_Validation_Files_2021-01-24.zip > SAMI_P0069_240121_tris_ValCert.pdf
+P0069,CC_ea578,94.1,
+P0069,CC_eb434,2284.0,
+P0069,CC_eb578,38676.0,
+P0069,CC_ind_off,0,
+P0069,CC_ind_slp,1,
+P0069,CC_psal,35,constant
```
Comment (batryn) [inline calibration/PHSEND/CGINS-PHSEND-P0069__20210124.csv:8]: Looks good

File: calibration/PHSEND/CGINS-PHSEND-P0068__20210122.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0068,CC_ea434,17372.0,source_file PHSEN-D_SAMI-pH_SN_P0068_Validation_Files_2021-01-22.zip > P0068 validation cert.pdf
+P0068,CC_ea578,94.1,
+P0068,CC_eb434,2284.0,
+P0068,CC_eb578,38676.0,
+P0068,CC_ind_off,0,
+P0068,CC_ind_slp,1,
+P0068,CC_psal,35,Constant
```
Comment (batryn) [inline calibration/PHSEND/CGINS-PHSEND-P0068__20210122.csv:8]: Looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0050__20210111.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0050,CC_cala,0.0088,source_file PCO2W-B_SAMI-CO2_SN_C0050_Calibration_Files_2021-01-11.zip > c50_010721E_CalCert.pdf
+C0050,CC_calb,0.8330,
+C0050,CC_calc,-1.8412,
+C0050,CC_calt,10.5457,
+C0050,CC_ea434,19706,constant
+C0050,CC_ea620,34,constant
+C0050,CC_eb434,3073,constant
+C0050,CC_eb620,44327,constant
```
Comment (batryn) [inline calibration/PCO2WB/CGINS-PCO2WB-C0050__20210111.csv:9]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01318__20210122.csv [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+1318,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1318_Calibration_Files_2021-01-22.zip > BBFL2W-1318 Charsheets.pdf
+1318,CC_dark_counts_cdom,43,
+1318,CC_dark_counts_chlorophyll_a,46,
+1318,CC_dark_counts_volume_scatter,48,
+1318,CC_depolarization_ratio,0.039,Constant
+1318,CC_measurement_wavelength,700,Constant
+1318,CC_scale_factor_cdom,0.0902,
+1318,CC_scale_factor_chlorophyll_a,0.0106,
+1318,CC_scale_factor_volume_scatter,2.405E-06,
+1318,CC_scattering_angle,124,Constant
```
Comment (batryn) [inline calibration/FLORTD/CGINS-FLORTD-01318__20210122.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01219__20210127.csv [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+1219,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1219_Calibration_Files_2021-01-27.zip > BBFL2W-1219 Charsheets.pdf
+1219,CC_dark_counts_cdom,48,
+1219,CC_dark_counts_chlorophyll_a,51,
+1219,CC_dark_counts_volume_scatter,43,
+1219,CC_depolarization_ratio,0.039,Constant
+1219,CC_measurement_wavelength,700,Constant
+1219,CC_scale_factor_cdom,0.0907,
+1219,CC_scale_factor_chlorophyll_a,0.0123,
+1219,CC_scale_factor_volume_scatter,1.919e-06,
+1219,CC_scattering_angle,124,Constant
```
Comment (batryn) [inline calibration/FLORTD/CGINS-FLORTD-01219__20210127.csv:11]: Looks good

File: calibration/ADCPTF/CGINS-ADCPTF-20530__20210204.csv [hunk 9]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+20530,CC_scale_factor1,0.45, source_file ADCPT-F_WHS150-1500_SN_20530_Configuration_Summary_2021-02-04.pdf
+20530,CC_scale_factor2,0.45,
+20530,CC_scale_factor3,0.45,
+20530,CC_scale_factor4,0.45,
```
Comment (batryn) [inline calibration/ADCPTF/CGINS-ADCPTF-20530__20210204.csv:5]: Looks good

**Review by batryn — APPROVED** [review]
Looks good

**Comment by snwhite** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/PCO2WB/CGINS-PCO2WB-C0050__20210111.csv (hunk 6 above)
Comment (asmith-whoi) [inline calibration/PCO2WB/CGINS-PCO2WB-C0050__20210111.csv:9]: OK

File: calibration/PHSEND/CGINS-PHSEND-P0068__20210122.csv (hunk 5 above)
Comment (asmith-whoi) [inline calibration/PHSEND/CGINS-PHSEND-P0068__20210122.csv:8]: OK

File: calibration/PHSEND/CGINS-PHSEND-P0069__20210124.csv (hunk 4 above)
Comment (asmith-whoi) [inline calibration/PHSEND/CGINS-PHSEND-P0069__20210124.csv:8]: OK

File: calibration/SPKIRB/CGINS-SPKIRB-00270__20201124.csv (hunk 3 above)
Comment (asmith-whoi) [inline calibration/SPKIRB/CGINS-SPKIRB-00270__20201124.csv:4]: OK

File: calibration/SPKIRB/CGINS-SPKIRB-00271__20201124.csv (hunk 2 above)
Comment (asmith-whoi) [inline calibration/SPKIRB/CGINS-SPKIRB-00271__20201124.csv:4]: OK

File: calibration/SPKIRB/CGINS-SPKIRB-00275__20201124.csv (hunk 1 above)
Comment (asmith-whoi) [inline calibration/SPKIRB/CGINS-SPKIRB-00275__20201124.csv:4]: OK

**Review by asmith-whoi — APPROVED** [review]

**Pull request merged**